x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/alert_actions.tsx

```diff
@@ -57,7 +55,7 @@ export function AlertActions({
       basePath: { prepend },
     },
   } = useMlKibana().services;
-  const casesPrivileges = cases?.helpers.canUseCases();
+  const casesPrivileges = cases?.helpers?.canUseCases() ?? false;
```

Comment:
If we isolate ml alerts to be used only for stack cases we should use `cases?.helpers?.canUseCases([CASE_APP_ID])`

Reply:
I think you need it because the `ml` plugin has the cases plugin as optional and can be disabled.